packages/plugins/appium-helper/README.md

```diff
@@ -35,16 +35,16 @@ import { AppiumDriver } from "@bam.tech/appium-helper";
 test("e2e", async () => {
   const driver = await AppiumDriver.create({
     // `npx @perf-profiler/profiler getCurrentApp` will display info for the current app
-    appPackage: com.example,
-    appActivity: com.example.MainActivity,
+    appPackage: "com.example",
```

Comment:
oh wow, can't believe this had been wrong for so long

Thanks for taking the time to update the readme as well!